{{cookiecutter.repo_name}}/compose/pycharm/.ssh_keys_to_docker/id_rsa

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
this is a feature. Because this debug container with sshd is used only for debbuging, we do not care, that everyone have access to it. Typing password all the time is just annoying.